frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -217,16 +214,14 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
             fieldId="pvc-size"
             className="co-restore-pvc-modal__input co-restore-pvc-modal__ocs-size"
           >
-            {snapshotClassResourceLoaded ? (
+            {!pvcStorageClassName && scResourceLoaded ? (
```

Comment:
```suggestion
            {!!pvcStorageClassName && scResourceLoaded ? (
```

Reply:
Sorry, i missed this to notice while testing as it create with same size in restore, i will update this